README.md

```diff
@@ -354,18 +355,18 @@ Rack::Attack.throttled_responder = lambda do |request|
   headers = {
     'RateLimit-Limit' => match_data[:limit].to_s,
     'RateLimit-Remaining' => '0',
-    'RateLimit-Reset' => (now + (match_data[:period] - now % match_data[:period])).to_s
+    'RateLimit-Reset' => (match_data[:retry_after] - now).to_s
   }
 
   [ 429, headers, ["Throttled\n"]]
 end
+Rack::Attack.configuration.throttled_responder_is_offset_aware = true
 ```
 
-
-For responses that did not exceed a throttle limit, Rack::Attack annotates the env with match data:
+For responses that exceeded a throttle limit, Rack::Attack annotates the env with match data:
```

Comment:
Actually, I think both of them are somehow inaccurate (or incomplete) 🤔. In `spec/rack_attack_throttle_spec.rb` we can see that `request.env['rack.attack.throttle_data'][name]` is always set (when using the throttling feature), except when the provided block returns `nil`.

Reply:
![image](https://github.com/rack/rack-attack/assets/6373536/b76c1cae-422f-448b-ab69-9b376397eb9a)

so maybe this is just some confusion between `throttle_data` ('rack.attack.throttle_data') and `matched_data` which include 'rack.attack.matched', 'rack.attack.match_discriminator', 'rack.attack.match_type', and 'rack.attack.match_data'.